sql/updates/latest-dev.sql

```diff
@@ -2,3 +2,4 @@ CREATE SCHEMA IF NOT EXISTS timescaledb_experimental;
 GRANT USAGE ON SCHEMA timescaledb_experimental TO PUBLIC;
 DROP FUNCTION IF EXISTS _timescaledb_internal.block_new_chunks;
 DROP FUNCTION IF EXISTS _timescaledb_internal.allow_new_chunks;
+DROP FUNCTION IF EXISTS _timescaledb_internal.refresh_continuous_aggregate;
```

Comment:
why does post-release change latest-dev?

Reply:
Because we are not defining the experimental schema in 2.3.1, which means that the function `_timescaledb_internal.refresh_continuous_aggregate` is still present in the `ddl_internals.sql` file. It should probably not have been removed in the previous commit, but since that was already present in the master branch and there were a significant rewrite of the release branch to handle the un-removal of PG11, this slipped through and was not caught by any tests.